ckanext/canada/templates/public/footer.html

```diff
@@ -6,7 +6,7 @@
 <li><a href="{{ _('http://news.gc.ca/') }}">{{ _('News') }}</a></li>
 <li><a href="{{ _('https://www.canada.ca/en/government/system/laws.html') }}">{{ _('Treaties, laws and regulations') }}</a></li>
 <li><a href="{{ _('https://www.canada.ca/en/transparency/reporting.html') }}">{{ _('Government-wide reporting') }}</a></li>
-<li><a href="{{ _('/en/user') }}">{{ _('Open Government Log In') }}</a></li>
```

Comment:
this template is for the public-facing site, we copy this template from the one Drupal uses which is /en/user/login